Source/Noesis.Javascript/JavascriptFunction.cpp

```diff
@@ -74,21 +74,28 @@ System::Object^ JavascriptFunction::Call(... cli::array<System::Object^>^ args)
 
 bool JavascriptFunction::operator==(JavascriptFunction^ func1, JavascriptFunction^ func2)
 {
-    bool func1_null = ReferenceEquals(func1, nullptr),
-         func2_null = ReferenceEquals(func2, nullptr);
-    if (func1_null != func2_null)
-		return false;
-    if (func1_null && func2_null)
+    if (ReferenceEquals(func1, func2))
         return true;
+    if (ReferenceEquals(func1, nullptr) != ReferenceEquals(func2, nullptr))
```

Comment:
is there any difference between this and simply comparing `(func1 == nullptr) != (func2 == nullptr)`?

Reply:
Probably not, I just didn't want to change the logic too much, so I just inlined the variables.